synapse/storage/databases/main/event_federation.py

```diff
@@ -114,6 +114,10 @@ def __init__(self, room_id: str):
 
 
 class EventFederationWorkerStore(SignatureWorkerStore, EventsWorkerStore, SQLBaseStore):
+    # TODO: this attribute comes from EventPushActionWorkerStore. Should we inherit from
+    # that store so that mypy can deduce this for itself?
+    stream_ordering_month_ago: Optional[int]
```

Comment:
I can't see _why_ this should live on EventPushActionsWorkerStore (and I can't see anywhere else that uses it). I think it'd make more sense to move this onto EventsWorkerStore if we anticipate other users, and here if not.

Reply:
`EventPushActionsWorkerStore` used to use it -- see #13118.

Reply:
(This change seems unrelated, did the above changes shake it out somehow?)

Reply:
If I don't make this change then mypy doesn't know what `self.stream_ordering_month_ago` is elsewhere in the file. That means it's treated as `Any`, which means the fix to the signature of `execute` doesn't spot the missing comma typo.

Reply:
> I think it'd make more sense to move this onto EventsWorkerStore if we anticipate other users, and here if not.

I might have overly aggressively resolved this -- did we want to do this?

Reply:
I wasn't in any particular hurry tbh!